panels/configspec.ini

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
I think the comments explain everything, but let me know if there are any questions.